README.md

```diff
@@ -12,4 +12,4 @@ I've made the following video to try to make my reasons for making this clear, b
 I don't hate AIs,
 I love humans!
 
-Check out my video here: https://www.youtube.com/watch?v=y3wVvDbJHrU
+Check out my video here: https://youtu.be/qjPAWy_2FL0
```

Comment:
ha! I fixed this in main too, but thanks for the driveby :sweat_smile:

Reply:
(code changes in another pr plz)

Reply:
sure thing.